src/Microsoft.OData.Core/ODataMessageReaderSettings.cs

```diff
@@ -104,10 +104,6 @@ public ValidationKinds Validations
         /// ODataMessageReaderSettings.BaseUri may be deprecated in the future, please use ODataMessageReaderSettings.baseUri instead.
         /// </summary>
         /// <returns>The base URI used in payload.</returns>
-        /// <remarks>
-        /// This URI will be used in ATOM format only, it would be overridden by &lt;xml:base /&gt; element in ATOM payload.
-        /// If the URI does not end with a slash, a slash would be appended automatically.

```

Comment:
Maybe return this line in the remarks

Reply:
Will it still have the atom format explanation?

Reply:
@ElizabethOkerio Not both lines. The second line: "If the URI does not end with a slash, a slash would be appended automatically."
I think it's relevant even for non-ATOM scenarios